pkg/addons/data/ClusterRole.rbac.authorization.k8s.io/dedicated-cluster-reader.yaml

```diff
@@ -0,0 +1,139 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  annotations:
+    authorization.openshift.io/system-only: 'true'
+  name: dedicated-cluster-reader
+rules:
+- apiGroups:
+  - ''
+  - user.openshift.io
+  attributeRestrictions: null
+  resources:
+  - groups
+  - users
+  - identities
+  - useridentitymappings
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ''
+  - authorization.openshift.io
+  attributeRestrictions: null
+  resources:
+  - rolebindings
+  - clusterrolebindings
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - rbac.authorization.k8s.io
```

Comment:
Similar comment as above, you can collapse this with the set above.

Reply:
done